project/profiles/capacity4more/modules/c4m/content/c4m_content/c4m_content.module

```diff
@@ -14,14 +14,14 @@
 function c4m_content_form_node_form_alter(&$form, &$form_state, $form_id) {
   // Checks if the form's content type is being used as a message,
   // to add a send notification option.
-  if (in_array($form_state['node']->type, c4m_message_valid_node_types())) {
```

Comment:
@anvmn 
if `c4m_message_valid_node_types()` was created especially for here and is not being used anywhere else, please remove it.